test/libsolidity/syntaxTests/viewPureChecker/gas_value_without_call.sol

```diff
@@ -1,13 +1,13 @@
 contract C {
     function f() external payable {}
 	function g(address a) external pure {
-		a.call.value(42);
```

Comment:
Here I would not change the syntax, but just keep both versions for now in the same test.